build.gradle

```diff
@@ -413,20 +566,50 @@ if (project.hasProperty('mavenUser') && project.hasProperty('jcecertAlias')) {
                 }
             }
         }
+    }
+
+    nexusPublishing {
         repositories {
-            maven {
-                url = 'https://aws.oss.sonatype.org/service/local/staging/deploy/maven2/'
-                credentials {
-                    username mavenUser
-                    password mavenPassword
-                }
+            sonatype {
+                nexusUrl = uri('https://aws.oss.sonatype.org/service/local/')
+                username = mavenUser
+                password = mavenPassword
             }
         }
     }
 
     signing {
         sign publishing.publications.mavenJava
     }
+
+    closeSonatypeStagingRepository {
+        doLast {
+            def idAsString = stagingRepositoryId.get()
+            def fw = new java.io.FileWriter("${buildDir}/fake_maven/staging.properties")
```

Comment:
do we need to write to both `CODEBUILD_SRC_DIR_Stage`'s build dir and `CODEBUILD_SRC_DIR_Stage_FIPS`'s build dir?

Reply:
Not explicitly. CodePipelines does that for us.